Week01/Problem01/apavlova_01.py

```diff
@@ -0,0 +1,15 @@
+#sum of all multiples of 3 and 5 below 1000
+#function for calculating the multiple of x up to the value y
+def MultSum(x,y):
+#x is used as a the multiplier, while the Mult variable is used to keeping
```

Comment:
I would encourage you to write [function docstrings](https://www.python.org/dev/peps/pep-0257/) instead of in-line comments to document your functions.  The biggest advantage of docstrings is that they can be queried anywhere your code is used by calling `help(MultSum)`, while in-line comments are only visible when looking at the code itself.